backend/DB/migrations/20201202101549_placement_hours_to_type.js

```diff
@@ -0,0 +1,15 @@
+
+exports.up = async function(knex) {
+    let hasTable = await knex.schema.hasTable("placements");
+    return hasTable ? knex.schema.table('placements', (table) => {
+        table.dropColumn('working_hours');
+        table.enum("employment_type", ["Full Time", "Part Time", "Contract", "Internship"]);
```

Comment:
I would suggest to call the different types in constant names, so that we avoid the space in between, and to be able to match them with constants if needed. I mean FULL_TIME, PART_TIME, CONTRACT, INTERNSHIP. What do you think?

Reply:
Ok